sonar-plugin-api/src/main/java/org/sonar/api/platform/ServerStartHandler.java

```diff
@@ -20,12 +20,14 @@
 package org.sonar.api.platform;
 
 import org.sonar.api.ExtensionPoint;
+import org.sonar.api.server.ComputeEngineSide;
 import org.sonar.api.server.ServerSide;
 
 /**
  * @since 2.2
  */
 @ServerSide
+@ComputeEngineSide
 @ExtensionPoint
 public interface ServerStartHandler {
```

Comment:
it's actually required. Some components such as the Views and DevCockpit starters rely on it